modules/distribution/src/repository/resources/dbscripts/apimgt/mssql.sql

```diff
@@ -649,14 +651,17 @@ CREATE TABLE AM_APPLICATION (
     APPLICATION_TIER VARCHAR(50) DEFAULT 'Unlimited',
     CALLBACK_URL VARCHAR(512),
     DESCRIPTION VARCHAR(512),
-	APPLICATION_STATUS VARCHAR(50) DEFAULT 'APPROVED',
-	CREATED_BY VARCHAR(100),
+    APPLICATION_STATUS VARCHAR(50) DEFAULT 'APPROVED',
+    GROUP_ID VARCHAR(100),
+    CREATED_BY VARCHAR(100),
     CREATED_TIME DATETIME,
     UPDATED_BY VARCHAR(100),
     UPDATED_TIME DATETIME,
+    UUID VARCHAR(256),
     FOREIGN KEY(SUBSCRIBER_ID) REFERENCES AM_SUBSCRIBER(SUBSCRIBER_ID) ON UPDATE CASCADE,
     PRIMARY KEY(APPLICATION_ID),
-    UNIQUE (NAME,SUBSCRIBER_ID)
+    UNIQUE (NAME,SUBSCRIBER_ID),
+    UNIQUE (UUID)
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04